### PR TITLE
Clean up Retry Service Text

### LINKED ISSUE #67: Packet Number in Retry Token
Currently the Initial Packet number MAY be encoded in the Retry Token. We must either:

- Include language that a server MUST NOT reject a token because this information is not present; OR
- Just make it part of the format. (IMO this would be bad, because then the Retry service would have to decrypt the packet number)

### PR DESCRIPTION
**Pull request by martinduke**
Fix #67 and #68

### CODE REVIEW

**Review by nibanks — APPROVED** [review]

File: draft-ietf-quic-load-balancers.md [hunk 1]
```diff
@@ -852,9 +873,8 @@ the server can issue an up-to-date token to the client.
 
 When issuing Retry or NEW_TOKEN tokens, the server MUST include the client IP
 address in the authenticated data as specified in
-{{token-protection-with-aead}}. It MUST include a means of distinguishing
-service-generated Retry tokens, server-generated Retry tokens (if different),
-and NEW_TOKEN tokens.
+{{token-protection-with-aead}}. The ODCIL and RSCIL fields are zero for
+NEW_TOKEN tokens, making them easily distinguishable from Retry tokens.
```
Comment (huitema) [inline draft-ietf-quic-load-balancers.md:878]: The text about port verification should probably be added there, or in the next paragraphs.

**Review by huitema — COMMENTED** [review]
Most changes appear editorial, except for the addition of the port number field. If you add that port, you need to also specify how it should be processed!

File: draft-ietf-quic-load-balancers.md [hunk 2]
```diff
@@ -621,7 +621,8 @@ has the following additional fields:
 
 ODCIL: The length of the original destination connection ID from the triggering
 Initial packet. This is in cleartext to be readable for the server, but
-authenticated later in the token.
+authenticated later in the token. The Retry Service SHOULD reject any token
+in which the value is less than 8.
```
[the remark this answers is not in the document]
Comment (martinduke) [reply draft-ietf-quic-load-balancers.md:626]: Retry services are version-specific, so this is OK.

**Comment by martinduke** [comment]
Thanks Christian,

Your comments ended up as an editorial rewrite of the section, with port
verification added.

**Pull request merged**